clippy_tests/examples/len_zero.stderr

```diff
@@ -46,45 +46,45 @@ error: length comparison to zero
    --> len_zero.rs:130:8
     |
 130 |     if x.len() == 0 {
-    |        ^^^^^^^^^^^^ help: using `is_empty` is more concise: `x.is_empty()`
+    |        ^^^^^^^^^^^^ help: using `is_empty` is more concise:: `x.is_empty()`
```

Comment:
I believe so. Removing the colon from https://github.com/rust-lang-nursery/rust-clippy/blob/master/clippy_lints/src/len_zero.rs#L176 gets rid of the duplicate.